ntc_templates/templates/cisco_ios_show_endpoint-tracker_tracker-group.textfsm

```diff
@@ -0,0 +1,13 @@
+Value NAME (\S+)
+Value TRACKERS (.+?)
+Value STATUS (\S+\(.*\))
+Value RTT ((?:\d+(?:,\s*\d+)*))
+Value PROBE_ID ((?:\d+(?:,\s*\d+)*))
+
+Start
+  ^\s*Tracker\s+Name\s+Element\s+trackers\s+name\s+Status\s+RTT\s+in\s+msec\s+Probe\s+ID -> Tracker
+
+Tracker
+  ^${NAME}\s+${TRACKERS}\s+${STATUS}\s+${RTT}\s+${PROBE_ID} -> Record
```

Comment:
@sjhloco
Not being cheeky here, for this simple template another state probably isn't necessary.
Thoughts?

```suggestion
  ^\s*Tracker\s+Name\s+Element\s+trackers\s+name\s+Status\s+RTT\s+in\s+msec\s+Probe\s+ID
  ^${NAME}\s+${TRACKERS}\s+${STATUS}\s+${RTT}\s+${PROBE_ID} -> Record
```